<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XDebuggerWidget</class>
 <widget class="QWidget" name="XDebuggerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonRun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonStepInto">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>SI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonStepOver">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>SO</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidgetMain">
     <property name="currentIndex">
      <number>8</number>
     </property>
     <widget class="QWidget" name="tabCPU">
      <attribute name="title">
       <string>CPU</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="DW_DisasmView" name="widgetDisasm" native="true"/>
          <widget class="DW_HexView" name="widgetHex" native="true"/>
         </widget>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QScrollArea" name="scrollAreaRegs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="DW_RegistersView" name="widgetRegs">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1118</width>
              <height>69</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="DW_StackView" name="widgetStack" native="true"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabActions">
      <attribute name="title">
       <string>Actions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonActionsRecord">
           <property name="text">
            <string>Record</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditActions">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLog">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBreakpoints">
      <attribute name="title">
       <string>Breakpoints</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabMemoryMap">
      <attribute name="title">
       <string>Memory map</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="DW_ProcessMemoryMap" name="widgetProcessMemoryMap" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCallStack">
      <attribute name="title">
       <string>Call stack</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabThreads">
      <attribute name="title">
       <string>Threads</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabHandles">
      <attribute name="title">
       <string>Handles</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabModules">
      <attribute name="title">
       <string>Modules</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="DW_ProcessModulesWidget" name="widgetProcessModules" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DW_HexView</class>
   <extends>QWidget</extends>
   <header>dw_hexview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DW_DisasmView</class>
   <extends>QWidget</extends>
   <header>dw_disasmview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DW_RegistersView</class>
   <extends>QWidget</extends>
   <header>dw_registersview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DW_StackView</class>
   <extends>QWidget</extends>
   <header>dw_stackview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DW_ProcessMemoryMap</class>
   <extends>QWidget</extends>
   <header>dw_processmemorymap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DW_ProcessModulesWidget</class>
   <extends>QWidget</extends>
   <header>dw_processmoduleswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
